migrations/20220805183250-create-resource.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
       },
       CategoryId: {
         type: Sequelize.INTEGER,
+        references: {
+          model: {
+            tableName: 'Categories',
+          },
+          key: 'id',
+        },
```

Comment:
This modification to the resource migration tells Sequelize which model/table the CategoryId references.

Doing this sets up a formal FOREIGN KEY relationship in the database. The Postgres database will now automatically check that any CategoryId value actually matches an id in the Categories table.

This will also PREVENT you from accidentally deleting a Category if it is being referenced by a Resource.

In SQL databases, this these features are called REFERENTIAL INTEGRITY. That is, the database makes sure that your references are valid, and cannot be broken.